src/Api/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+WORKDIR /src
+COPY ["src/Api/Api.csproj", "src/Api/"]
+COPY ["src/Service/Service.csproj", "src/Service/"]
+COPY ["src/Common/Common.csproj", "src/Common/"]
+RUN dotnet restore "src/Api/Api.csproj"
+COPY . .
+WORKDIR "/src/src/Api"
+RUN dotnet build "Api.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "Api.csproj" -c Release -o /app/publish /p:UseAppHost=false
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app/publish .
+ENTRYPOINT ["dotnet", "Passwordless.Api.dll"]

```

Comment:
Could probably use the `Api.dockerfile` in root, right?
Also, form what I understand, you don't actually use this image?